### ISSUE TITLE
How to cite the original paper

### ISSUE THREAD

**Issue by ugurdi** [matterport/Mask_RCNN#1498]
Sorry if I missed in the README but where is the original academic paper to cite?

**Comment by chAwater** [comment]
Please click the link in the first text line of README:
> This is an implementation of [Mask R-CNN](https://arxiv.org/abs/1703.06870) on Python 3, Keras, and TensorFlow.

**Comment by ugurdi** [comment]
Thank you very much. My bad.
